rocky/crisis_room/models.py

```diff
@@ -17,19 +29,32 @@ def __str__(self) -> str:
         return super().__str__()
 
 
+MIN_POSITION = 1
+MAX_POSITION = 16
+
+
+def get_default_dashboard_data_settings() -> dict[str, Any]:
+    return {"size": "1", "columns": {}}
+
+
 class DashboardData(models.Model):
     dashboard = models.ForeignKey(Dashboard, on_delete=models.CASCADE, null=True)
-    recipe = models.UUIDField(blank=False, null=True)
+    name = models.CharField(blank=True, null=True, max_length=126)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    recipe = models.UUIDField(blank=True, null=True)
+    query_from = models.CharField(blank=True, max_length=32, null=True)
+    query = models.CharField(blank=True, null=True)
```

Comment:
```suggestion
    query_from = models.CharField(blank=True, max_length=32)
    query = models.CharField(blank=True)
```
Null is not needed on string fields.